pytwis/pytwis_clt.py

```diff
@@ -593,28 +532,99 @@ def pytwis_cli():
 
     try:
         if len(args.socket) > 0:
-            twis = Pytwis(socket=args.socket, db=args.db, password=args.password)
+            pytwis = Pytwis(socket=args.socket, db=args.db, password=args.password)
         else:
-            twis = Pytwis(hostname=args.hostname, port=args.port, db=args.db, password=args.password)
+            pytwis = Pytwis(hostname=args.hostname, port=args.port, db=args.db, password=args.password)
+        return pytwis, prompt
```

Comment:
Where do we set "prompt"?

Reply:
Line 520. 
    if len(args.socket) > 0:
        print('The input Redis server socket is {}'.format(args.socket))
        prompt = args.socket
    else:
        print('The input Redis server hostname is {}.'.format(args.hostname))
        print('The input Redis server port is {}.'.format(args.port))
        prompt = '{}:{}'.format(args.hostname, args.port)

Reply:
Sorry. My bad. I just looked at the diff part and failed to expand the common part. Now I see it.